test/ClassBuilder/CallTest.js

```diff
@@ -40,7 +40,7 @@ test('ClassBuilder:CallTest', function(assert) {
     C2(2);
     var c2a = new C2();
     assert.strictEqual(C2(), 2, '__call returns this.a (2)');
-    assert.strictEqual(c2.getA(), 1, 'c2.getA() returns this.a (2)');
+    assert.strictEqual(c2.getA(), 1, 'c2.getA() returns this.a (2)'); // these are inconsistent. Either the output comment is wrong (2), or the test is wrong (1). Which was intended?
```

Comment:
Nice catch!

So, there's actually 2 things wrong here. Actually one, but the other isn't implemented afaik.

1. The test is correct. The comment is wrong.
2. The "static" modifier isn't implemented. If I remember correctly, I couldn't find an efficient way to share a non-object variable over multiple instances of classes.

A static call function (__call), should NOT be able to overwrite the value of non-static properties in instantiated classes. However, If a property is declared as `static`, then it is allowed. Currently this isn't possible in JOII.